lib/SILOptimizer/PassManager/PassPipeline.cpp

```diff
@@ -635,6 +635,9 @@ SILPassPipelinePlan SILPassPipelinePlan::getOnonePassPipeline() {
   // Has only an effect if the -gsil option is specified.
   P.addSILDebugInfoGenerator();
 
+  // Finally serialize the SIL if we are asked to.
+  P.addSerializeSILPass();
```

Comment:
If you're adding this here, don't you need to remove an explicit `serialize` call from the Frontend?

Reply:
No. This can be seen by noting that the perf pipeline does the exact same thing without issue.

Reply:
Jordan and I spoke offline. He agreed with me that the sib problem is out of scope and I should file an SR for this. We agreed on the changes he wants in the frontend. I am going to update the PR.